librz/diff/unified_diff.c

```diff
@@ -176,12 +382,22 @@ RZ_API RZ_OWN char *rz_diff_unified_text(RZ_NONNULL RzDiff *diff, RZ_NULLABLE co
 				diff_unified_append_data(diff, diff->a, op->a_beg, op->a_end, sb, ' ', color);
 				continue;
 			}
-			if (op->type == RZ_DIFF_OP_DELETE || op->type == RZ_DIFF_OP_REPLACE) {
+			if (op->type == RZ_DIFF_OP_DELETE) {
 				diff_unified_append_data(diff, diff->a, op->a_beg, op->a_end, sb, '-', color);
 			}
-			if (op->type == RZ_DIFF_OP_INSERT || op->type == RZ_DIFF_OP_REPLACE) {
+			if (op->type == RZ_DIFF_OP_INSERT) {
 				diff_unified_append_data(diff, diff->b, op->b_beg, op->b_end, sb, '+', color);
 			}
+			if (op->type == RZ_DIFF_OP_REPLACE) {
```

Comment:
How about adding else? `if delete else if insert else if replace` ? we do not need to be if if if since that replace has its own scope

Reply:
Ok will do next.

Reply:
Done in #1177.